diff --git a/node_modules/cordova-plugin-file/types/index.d.ts b/node_modules/cordova-plugin-file/types/index.d.ts
index aeb3e30..4eb017c 100644
--- a/node_modules/cordova-plugin-file/types/index.d.ts
+++ b/node_modules/cordova-plugin-file/types/index.d.ts
@@ -17,7 +17,7 @@ interface Window {
     requestFileSystem(
         type: LocalFileSystem,
         size: number,
-        successCallback: (fileSystem: FileSystem) => void,
+        successCallback: (fileSystem: CordovaFileSystem) => void,
         errorCallback?: (fileError: FileError) => void): void;
     /**
      * Look up file system Entry referred to by local URL.
@@ -42,7 +42,7 @@ interface Window {
 }
 
 /** This interface represents a file system. */
-interface FileSystem {
+interface CordovaFileSystem {
     /* The name of the file system, unique across the list of exposed file systems. */
     name: string;
     /** The root directory of the file system. */
@@ -63,7 +63,7 @@ interface Entry {
     /** The full absolute path from the root to the entry. */
     fullPath: string;
     /** The file system on which the entry resides. */
-    filesystem: FileSystem;
+    filesystem: CordovaFileSystem;
     nativeURL: string;
     /**
      * Look up metadata about this entry.
